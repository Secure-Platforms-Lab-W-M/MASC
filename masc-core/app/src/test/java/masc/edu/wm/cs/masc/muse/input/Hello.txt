
package masc.edu.wm.cs.masc.muse.input;

class Hello{

    public static void main(String[] args){
        System.out.println("Hello World!");
    }

}