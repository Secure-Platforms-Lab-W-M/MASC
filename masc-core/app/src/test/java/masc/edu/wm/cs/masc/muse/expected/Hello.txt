
package masc.edu.wm.cs.masc.muse.input;

class Hello{

    public static void main(String[] args){
        System.out.println("This is a mutation");
        System.out.println("Hello World!");
    }

}